torchaudio/datasets/tedlium.py

```diff
@@ -35,8 +35,8 @@
         "url": "http://www.openslr.org/resources/51/TEDLIUM_release-3.tgz",
         "checksum": "ad1e454d14d1ad550bc2564c462d87c7a7ec83d4dc2b9210f22ab4973b9eccdb",
         "data_path": "data/",
-        "subset": None,
-        "supported_subsets": [None],
+        "subset": "train",
+        "supported_subsets": ["train", "test", "dev"],
```

Comment:
can you update the docstring [here](https://github.com/pytorch/audio/pull/2157/files#diff-51f17463716a26914065a17b11aabcb5ced2c0e83967e0445c1250ee7758e851R54) to reflect this